test/e2e/test_e2e_release.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
+# SPDX-License-Identifier: MIT
+
+set -euo pipefail
+
+echo "Starting e2e release test..."
+
+TEMP_VENV=$(mktemp -d)
+echo "Testing in temporary environment: $TEMP_VENV"
+
+cleanup() {
+    rm -rf "$TEMP_VENV" test-files/
+}
+
+trap cleanup EXIT
+
+echo "Creating fresh virtual environment"
+pushd "$TEMP_VENV"
+python3 -m venv test-env
+# shellcheck disable=SC1091
+source test-env/bin/activate
+
+echo "Installing built package..."
+popd > /dev/null
+python3 -m pip install dist/*.whl
```

Comment:
May it be a good idea to pass `dist` directory as a parameter to this script? I mean, now this `test_e2e_release.sh` needs to know that the wheel package is in `dist` folder. It can either be done implicitly (as it is now), or explicitly (as a parameter).